src/git/util/gitcommand.ts

```diff
@@ -42,7 +42,7 @@ export function getGitCommand(): string {
     }
 }
 
-export async function getOriginOfActiveFile(): Promise<string> {
+export async function getOriginOfActiveFile(remoteName: string): Promise<string> {
```

Comment:
Could you please reformat this to the following?

```ts
export async function getOriginOfActiveFile(
    remoteName: string,
): Promise<string> {
```

It is currently the only thing preventing the pipeline form succeeding.